src/main/java/com/autotune/analyzer/services/DSMetadataService.java

```diff
@@ -283,4 +301,108 @@ private Gson createGsonObject() {
     private boolean isValidBooleanValue(String value) {
         return value != null && (value.equals("true") || value.equals("false"));
     }
+
+    /**
+     * TODO temp solution to delete metadata, Need to evaluate use cases
+     *
+     * @param request
+     * @param response
+     * @throws ServletException
+     * @throws IOException
+     */
+    @Override
+    protected void doDelete(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
```

Comment:
Add javadoc for each of the APIs and mention the valid parameters that can be expected

Reply:
Added Javadoc for all metadata related APIs